bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/handler/ZonePlayerHandler.java

```diff
@@ -2827,27 +2833,41 @@ private void handleNotifForSharedQueue(@Nullable String currentQueueURI, Command
      */
     private void handleNotificationSound(Command notificationURL, ZonePlayerHandler coordinator)
             throws InterruptedException {
+        logger.debug("handleNotificationSound: URL {} coordinator {}", notificationURL, coordinator.getUDN());
         boolean sourceStoppable = !isPlayingOpticalLineIn(coordinator.getCurrentURI());
         String originalVolume = (isAdHocGroup() || isStandalonePlayer()) ? getVolume() : coordinator.getVolume();
         if (sourceStoppable) {
+            logger.debug("Stop current playback...");
             coordinator.stop();
             coordinator.waitForNotTransportState(STATE_PLAYING);
+            logger.debug("Current playback stopped");
+            logger.debug("Apply notification volume");
             applyNotificationSoundVolume();
         }
         long notificationPosition = coordinator.getQueueSize() + 1;
+        logger.debug("Add notification URI {} to queue", notificationURL);
         coordinator.addURIToQueue(notificationURL.toString(), "", notificationPosition, false);
+        logger.debug("Set current URI to {}", QUEUE_URI + coordinator.getUDN() + "#0");
```

Comment:
I'd suggest to either introduce a local variable with `QUEUE_URI + coordinator.getUDN() + "#0"` or wrap in conditional `if (logger.isDebugEnabled())`, so that the string is only constructed twice when debug level logging is enabled.